css/style.css

```diff
@@ -16,4 +12,102 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+ body {
+    font-family: 'Roboto', sans-serif;
+    -webkit-font-smoothing: antialiased;
+}
+/** navigation bar*/
+.top-navigation {
+    background-color: #fff;
+    position: fixed;
+    width: 100%;
+    border-bottom: 1px solid #DDD;
+}
+.top-navigation > .content {
+    height: 60px;
+}
+.top-navigation .logo {
+    height: 100px;
+}
+.top-navigation .links {
+    float: right;
+    color: grey;
+    font-size: 20px;
+    margin-top: 10px;
+}
+.top-navigation .links li {
+    display: inline-block;
+    margin-left: 1.563rem;
+}
+.top-navigation .links li.active {
+    color: #4c5058;
+    font-weight: 500;
+}
+.top-navigation .links li:hover {
+    cursor: pointer;
+    color: #333333;
+}
+/*
+Section-1
+**/
+.section-1{
+    background-image:url('../img/first-background.jpg');
+    background-position:center-bottom;
+    background-size:cover;
+    height:600px;
+    overflow: auto;
+}
+.section-1 h1{
+    color:white;
+    text-align:center;
+    font-size: 80px;
+}
+.section-1 h2{
+    color:white;
+    text-align:center;
+    font-size:40px;
+}
+
+
+/**
+the button
+*/
+.buttons{
+    color:white;;
+    text-align:center;
+    margin-top:1.563rem;
+}
```

Comment:
Here you could have styled the `button` element directly, instead of the parent element `.buttons`.
Can you add a `background-color` to the button so it looks like the original?